lib/botRepository.js

```diff
@@ -30,11 +30,12 @@ class BotRepository {
   }
 
   getLastPolledDate() {
-    return Rx.Observable.fromPromise(this.redis.get(LAST_POLLED_DATE));
+    return Rx.Observable.fromPromise(this.redis.get(LAST_POLLED_DATE))
+      .select(date => new Date(parseInt(date, 10)));
```

Comment:
I wonder why in JS they call `map` `select` and `flatMap` `selectMany` :( that's so confusing